jenkins/scality-glance-store-functional-tests/10-install-ring.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -xue
+
+git submodule init
```

Comment:
This should be done by Jenkins already. If not, it should be configured to do so. Same for the `update`.